src/main/scala/org/constellation/ConstellationNode.scala

```diff
@@ -197,7 +198,7 @@ class ConstellationNode(val configKeyPair: KeyPair,
 
   logger.info(s"Node init with API $httpInterface $httpPort peerPort: $peerHttpPort")
 
-  implicit val timeout: Timeout = Timeout(timeoutSeconds, TimeUnit.SECONDS)
+  constellation.StandardTimeout = Timeout(timeoutSeconds, TimeUnit.SECONDS)
```

Comment:
Ah you're right I changed it to a var

We have other constants that are supposed to follow this:

https://docs.scala-lang.org/style/naming-conventions.html

> Constant names should be in upper camel case. Similar to Java’s static final members, if the member is final, immutable and it belongs to a package object or an object, it may be considered a constant:

I always forget this ^